control-plane/gateways/deployment_init_container.go

```diff
@@ -143,6 +143,22 @@ func initContainer(config GatewayConfig, name, namespace string) (corev1.Contain
 			Value: consulNamespace,
 		})
 
+	if config.TLSEnabled {
+		container.Env = append(container.Env,
+			corev1.EnvVar{
+				Name:  "CONSUL_USE_TLS",
+				Value: "true",
+			},
+			corev1.EnvVar{
+				Name:  "CONSUL_CACERT_PEM",
+				Value: config.ConsulCACert,
+			},
+			corev1.EnvVar{
+				Name:  "CONSUL_TLS_SERVER_NAME",
+				Value: config.ConsulTLSServerName,
+			})
```

Comment:
Do these Name values exist as constants in the code already? If so, it would make sense to use them. If not, maybe they should?

Reply:
done and done with https://github.com/hashicorp/consul-k8s/pull/3429/commits/4b4fccda4960f9f5149597fc026ada4be52e7804